user_docs/en/changes.md

```diff
@@ -13,6 +13,7 @@
 
 ### Bug Fixes
 
+* Fixed problem with multiple expressions on a line in word documents: everything after the first expression was dropped (not spoken or brailled)
```

Comment:
please also add a reference to the PR number, and your GitHub username if you wish

```suggestion
* Fixed bug with multiple math expressions on the same line in Microsoft Word documents: everything after the first expression was not spoken or brailled. (#18526, @NSoiffer)
```